### PR TITLE
Bug 1119633 - Add Send SS, Send USSD and Send SMS default message.

### PR DESCRIPTION
**Pull request by weilonge**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[weilonge Sean Lee (weilonge)](https://github.com/weilonge) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1d96ce1bbcd1)

File: apps/system/locales/system.en-US.properties [hunk 1]
```diff
@@ -516,6 +516,9 @@ icc-message-maintitle=Operator message
 icc-message-subtitle=SIM {{id}}
 icc-confirmCall-defaultmessage=Call {{number}}?
 icc-alertMessage-defaultmessage=Sending message
```
Comment (frsela) [inline apps/system/locales/system.en-US.properties]: Check if this message (icc-alertMessage-defaultmessage) is used somewhere, if not used anywhere, please remove it

**Comment by try-server-hook** [comment]
[weilonge Sean Lee (weilonge)](https://github.com/weilonge) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5f1464a0441e)

**Comment by try-server-hook** [comment]
[weilonge Sean Lee (weilonge)](https://github.com/weilonge) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2acf668401b8)

**Pull request merged**